wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,8 +74,8 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
```

Comment:
Should we?
1) Never write a schema?  Does it need that schema during the processing of structure migrations (regular rails type migrations)?
2) Never write a gem file (probably, so if we do write one, it must be in the deck?)

Is it ok to install a link for db/migrations if not present in the deck?  If it is present, we could copy/link files from the gems.